configuration/analyzer_config.json

```diff
@@ -1749,6 +1749,32 @@
       }
     }
   },
+  "IPApi": {
+    "type": "observable",
+    "python_module": "ipapi.IPApi",
+    "description": "Gives information about IPs and DNS",
```

Comment:
please add link to external service with markdown

Reply:
@mlodic can you review it, please?